.github/workflows/release.yaml

```diff
@@ -9,12 +9,35 @@ jobs:
   release:
     runs-on: ubuntu-latest
     timeout-minutes: 60
+    strategy:
+      matrix:
+        node-version: ['lts/*']
```

Comment:
I used this because the `node.yaml` workflow uses it. However, it does currently throw some warnings in the log because we have our package.json set to require node `>= 18.0.0`, but LTS is v16.x. I'm not sure if we are actually using anything that really requires us to be on 18.x though.

Reply:
We can target v16 in the packages.json to clear that up.

Reply:
v18 is LTS in 3 months, so we could also just ignore the warnings for a bit :)

Reply:
Yeah this looks great!  I had just preemptively pushed some real basic workflows to build on. iirc I disabled them on the repo because I wasn’t sure about the PAT. I probably should have opened a pr to open discussion 😬

Reply:
I didn't know 18 was coming so soon, then ya let's keep it!